lib/src/components/molecules/star_rating.dart

```diff
@@ -162,11 +162,11 @@ extension on _StarValue {
   IconData icon({required bool rounded}) {
     switch (this) {
       case _StarValue.empty:
-        return rounded ? ZetaIcons.star_outline_round : ZetaIcons.star_outline_sharp;
+        return rounded ? Icons.star_outline_rounded : Icons.star_outline_sharp;
```

Comment:
Why are you not using the Zeta Icons here?

Reply:
updated with Zeta